pkg/agent/agent.go

```diff
@@ -294,10 +293,8 @@ func (i *Initializer) initInterfaceStore() error {
 	}
 	parseTunnelInterfaceFunc := func(port *ovsconfig.OVSPortData, ovsPort *interfacestore.OVSPortConfig) *interfacestore.InterfaceConfig {
 		intf := noderoute.ParseTunnelInterfaceConfig(port, ovsPort)
-		if intf != nil && port.OFPort == config.DefaultTunOFPort &&
-			intf.InterfaceName != i.nodeConfig.DefaultTunName {
-			klog.Infof("The discovered default tunnel interface name %s is different from the default value: %s",
-				intf.InterfaceName, i.nodeConfig.DefaultTunName)
+		if intf != nil && intf.InterfaceName != i.nodeConfig.DefaultTunName {
```

Comment:
This is incorrect because we also call this function on the ipsec tunnel port, only compare on the interface name may impact on the ipsec case.

Reply:
I see, the problem is that the function was using the port number as the condition for that, which is misleading and even potentially invalid. I will update that to use the interface type instead.

Reply:
Thanks for pointing it out, I fixed the logic